### DISCUSSION TITLE
Multi-coloured Lamps

### DISCUSSION THREAD

**Discussion by jordan8396** [MrTJP/ProjectRed#1721] [Ideas]
An idea I would love to see is a lamp block which lights up the colour of the input of a connected bundled cable.

So if I connect the lamp to a bundled cable & then power the red wire, the lamp goes red or if the yellow wire is powered then it turns yellow. I haven't thought of the exact logic of what happens if 2 inputs are received but maybe either the strongest signal is used, or the most recently changed signal?

My primary usecase for this would be for a compact traffic-light style system and compact disco/studio lights system.

**Comment by MrTJP** [comment]
The Illumar Smart Lamp (added in https://github.com/MrTJP/ProjectRed/pull/1855) has all the features you need :)
